scripts/copy_clinvar.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# this should be run with high-core low memory resources
+# e.g.
+# analysis-runner analysis-runner --cpu 8 --memory 2Gi scripts/copy_clinvar.sh 2022-09-17
```

Comment:
Should we try `--cpu 16 --memory lowmem`?

Reply:
updated the docstring

https://hail.is/docs/batch/api/batch/hailtop.batch.job.Job.html#hailtop.batch.job.Job.memory